awx/main/tests/unit/commands/test_dump_auth_config.py

```diff
@@ -0,0 +1,124 @@
+from io import StringIO
+import json
+from django.core.management import call_command
+from django.test import TestCase, override_settings
+
+
+# SAML
+@override_settings(SOCIAL_AUTH_SAML_SP_ENTITY_ID="SP_ENTITY_ID")
```

Comment:
this is nitpicking, but I think you can create a dictionary of all of these settings like
```python
settings_dict = {
SOCIAL_AUTH_SAML_SP_ENTITY_ID="SP_ENTITY_ID",
..
}
```
then call

```python
@override_settings(**settings_dict)
class TestDumpAuthConfigCommand(TestCase):
```

Reply:
yeah, that's much cleaner. reformatted the settings